### PR TITLE
DBZ-3027 Add Debezium UI image

### PR DESCRIPTION
**Pull request by jpechane**
https://issues.redhat.com/browse/DBZ-3027

### CODE REVIEW

File: ui/1.5/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    && chown 1001:root /deployments \
+    && curl https://repo1.maven.org/maven2/io/fabric8/run-java-sh/${RUN_JAVA_VERSION}/run-java-sh-${RUN_JAVA_VERSION}-sh.sh -o /deployments/run-java.sh \
+    && chown 1001 /deployments/run-java.sh \
+    && chmod 540 /deployments/run-java.sh \
+    && echo "securerandom.source=file:/dev/urandom" >> /etc/alternatives/jre/lib/security/java.security
+
+# Configure the JAVA_OPTIONS, you can add -XshowSettings:vm to also display the heap size.
+ENV JAVA_OPTIONS="-Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager"
+
+COPY --from=builder --chown=1001 /sources/backend/target/quarkus-app /deployments/
```
Comment (rk3rn3r) [inline ui/1.5/Dockerfile:54]: For better caching of independent layers Quarkus once updated the default Dockerfile to have 4 layers here. Does it make sense to do this here as well? Compare to the Dockerfile in the UI repo: https://github.com/debezium/debezium-ui/blob/master/backend/src/main/docker/Dockerfile#L58#L61

Comment (jpechane) [reply ui/1.5/Dockerfile:54]: @rk3rn3r I intentionally removed it as I think it is not important for Docker Hub deployed and built images

Comment (gunnarmorling) [reply ui/1.5/Dockerfile:54]: Hum, the lib caching is pretty good, no? I'd like that better, agreeing with @rk3rn3r here.

Comment (jpechane) [reply ui/1.5/Dockerfile]: Done

Comment (gunnarmorling) [reply ui/1.5/Dockerfile]: @jpechane, it's the same timeout now we see on GH Actions. I suppose the right Maven options need to be added?

Comment (rk3rn3r) [reply ui/1.5/Dockerfile]: From my experiences at my last job that timeout issue could be a docker-in-docker issue? Is travis using Docker?
Maybe we can switch to buildah or something that doesn't use docker deamon for building? On kubernetes such service containers often have low specs which lead to sometimes slow and unreliable builds. At least that was the case for us.

**Pull request merged**